Libraries/JavaScriptAppEngine/Initialization/InitializeJavaScriptAppEngine.js

```diff
@@ -25,11 +25,11 @@
 require('regenerator/runtime');
 
 if (typeof GLOBAL === 'undefined') {
-  GLOBAL = this;
+  global.GLOBAL = this;
```

Comment:
`GLOBAL` is valid in Node so removing it will probably break some third party npm modules.